docs/UserGuide.md

```diff
@@ -215,7 +215,7 @@ Examples:
 
 #### 3.2.4 Listing all students: `list`
 
-You can view the list of all students in **Reeve**.
+You can view the list of all students in **Reeve** in order sorted by the name of the student..
```

Comment:
A better way to phrase this would be "list of all students [...] sorted by name".